app/controllers/tweet_controller.rb

```diff
@@ -0,0 +1,87 @@
+class TweetsController < ApplicationController
+
+
+	get '/tweets' do
+		if logged_in?
+			@user = current_user
+			success
+			erb :'/tweets/tweets'
+		else
+			please_login
+			redirect '/login'
+		end
+	end
+
+
+	get '/tweets/new' do
+		if logged_in?
+			@user = current_user
+			erb :'/tweets/create_tweet'
+		else
+			please_login
+			redirect '/login'
+		end
+	end
+
+	post '/tweets/new' do
+		if filled_out(params)
+			@tweet = Tweet.create(content: params[:content])
+			@tweet.id = Tweet.all.last.id
+			@tweet.user_id = current_user.id
```

Comment:
The tweet creation and user association can both be accomplished in the same step:

``` ruby
@tweet = Tweet.create(content: params[:content], user: current_user)
```